### PR TITLE
docs: network upgrade

### PR DESCRIPTION
**Pull request by sammy1991106**

### CODE REVIEW

File: UPGRADE.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+3. Reset blockchain state
+```
+axelard unsafe-reset-all
+```
+
+
+4. Wait for the Axelar team to publish the new genesis file for the new chain. The genesis files can be found at
+- testnet: https://axelar-testnet.s3.us-east-2.amazonaws.com/genesis.json
+
+Once the new genesis file is published, place it in `/root/.axelard/config/`. **Note that the path may be different if you are running your node with the binaries.**
```
Comment (ggutoski) [inline UPGRADE.md:24]: is this in the docker container?  users no longer have permissions for `/root`

File: UPGRADE.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+1. Wait for the proposed upgrade block. Your node will panic and stop once the block is reached. If you are running a validator, you will need to vote for the upgrade proposal beforehand. You can do so by running
+```
+axelard tx gov vote ${proposal_id} yes \
+	--from validator --gas auto --gas-adjustment 1.5
+```
+
+2. Backup the state and keys.
+```
+mv /home/axelard/.axelar /home/axelard/.axelar_backup
```
Comment (ggutoski) [inline UPGRADE.md:11]: why `mv` and not `cp`?

Comment (talalashraf) [inline UPGRADE.md:11]:
```suggestion
mv $ROOT_DIRECTORY/.axelar $ROOT_DIRECTORY/.axelar_backup # run from outside the container
```

Comment (ggutoski) [reply UPGRADE.md:11]: Are people supposed to have a `ROOT_DIRECTORY` env var outside the container?  I don't have one on my node.

**Review by ggutoski — APPROVED** [review]

**Pull request merged**